### PR TITLE
Add IDs and pagination gear to Bitstream Formats

### LINKED ISSUE #1767: Display all bitstream formats on one page and include IDs.
**Is your feature request related to a problem? Please describe.**
DSpace 7 Bitstream Format Registry e.g., https://demo7.dspace.org/admin/registries/bitstream-formats, only displays 20 bitstream formats per page. Viewing the list on Demo with 81 formats takes 5 page clicks and a good memory.

**Describe the solution you'd like**
DSpace 6 XMLUI displays all the formats one page, e.g. https://demo.dspace.org/xmlui/admin/format-registry. This is preferable for several needs:

- Comparing related formats, for instance all the video formats. Paging may cut related formats into two pages.
- Searching for a term with a browser search
- Copying the list for a spreadsheet

Also, DSpace XMLUI includes a column for the format ID. This is a very useful reference for composing SQL queries, so I would like it included in the page.
DSpace6_XMLUI_bistream_registry

**Describe alternative

s or workarounds you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yingjin**
## References
_Add references/links to any related issues or PRs. These may include:_
* Fixes #1767
* Requires DSpace/DSpace#[pr-number] (if a REST API PR is required to test this)

## Description
Add ID column to Bitstream format Registry
Add Pagination gear
Add admin.registries.bitstream-formats.table.id to all i18n json5

## Instructions for Reviewers
Please add a more detailed description of the changes made by your PR. At a minimum, providing a bulleted list of changes in your PR is helpful to reviewers.

List of changes in this PR:
* First, ...
* Second, ...

**Include guidance for how to test or review your PR.** This may include: steps to reproduce a bug, screenshots or description of a new feature, or reasons behind specific changes. 

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [x ] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [ ] My PR passes [TSLint](https://palantir.github.io/tslint/) validation using `yarn run lint`
- [ ] My PR doesn't introduce circular dependencies
- [ ] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [ ] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new, third-party dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.

### CODE REVIEW

**Comment by jtimal** [comment]
Hi @yingjin ,

The IDs are correct, I have a problem in the listings when I add the option show 50 only 40 are displayed. I guess it's not a problem since the default is 81.
image
image
then I added elements to show 101 and the problem remained, however in the results of 100 the data are already correct.
image

**Review by tdonohue — CHANGES_REQUESTED** [review]
@yingjin :  Thanks for this contribution.  I'm seeing similar problems to @jtimal above.  The IDs display properly, but the pagination gear isn't working as expected:

* By default, the pagination gear shows that I should have 25 results. But, the default display shows only 20
* If I change the pagination gear to 50 results, the display updates but now it's only showing 40.

Additionally, I see that we are currently receiving some test failures in `bitstream-formats.component.spec.ts` based on these updates.  So, you'll need to update those tests to get them to pass again.  You can run `yarn test` locally to see those failures on your local machine.

**Comment by yingjin** [comment]
Sorry that I forgot to run the test and lint before the push. I hope this
time it will pass all the tests.

Best,
Ying

File: src/app/admin/admin-registries/bitstream-formats/bitstream-formats.component.ts [hunk 1]
```diff
@@ -33,25 +33,26 @@ export class BitstreamFormatsComponent implements OnInit, OnDestroy {
    * The current pagination configuration for the page used by the FindAll method
    * Currently simply renders all bitstream formats
    */
-  config: FindListOptions = Object.assign(new FindListOptions(), {
-    elementsPerPage: 20
-  });
+  // config: FindListOptions = Object.assign(new FindListOptions(), {
+  //   elementsPerPage: 20
+  // });
```
Comment (tdonohue) [inline src/app/admin/admin-registries/bitstream-formats/bitstream-formats.component.ts:38]: Small thing, can we just remove this unused code?  It doesn't appear there is a reason to keep this around. (and the comment just above it).

**Review by tdonohue — APPROVED** [review]
👍 Thanks @yingjin !  This works well & the code looks good to me. But, I have a small suggestion inline below to remove some unused code before this is merged.  If you could make that small update (or respond to my comment), then I think this is ready to merge soon.

File: src/app/admin/admin-registries/bitstream-formats/bitstream-formats.component.ts (hunk 1 above)
Comment (jtimal) [reply src/app/admin/admin-registries/bitstream-formats/bitstream-formats.component.ts:38]: Hi @yingjin 
I'm still working with the page layout, greetings!
image

**Comment by yingjin** [comment]
No problem. I will remove that unused code.

**Review by tdonohue — APPROVED** [review]
👍 Thanks @yingjin , this looks good to me now!

**Pull request merged**